gst/tensor_decoder/tensordec.c

```diff
@@ -740,11 +740,12 @@ gst_tensordec_label_set_output (GstTensorDec * self, GstBuffer * outbuf,
   GstMemory *out_mem;
   GstMapInfo out_info;
 
+  gst_buffer_set_size (outbuf, 0);
   g_assert (gst_buffer_get_size (outbuf) == 0);
 
   len = strlen (label);
 
-  out_mem = gst_allocator_alloc (NULL, (len + 1), NULL);
+  out_mem = gst_allocator_alloc (NULL, len, NULL);
```

Comment:
Then isn't it invalid?:
```
   strncpy ((char *) out_info.data, label, len);
   ((char *) out_info.data)[len] = '\0';
```

Reply:
regarding this, I tested with gst-launch test cmd,
original code adds to string with garbage datas,
but with newly changed code, it makes proper text output
so I changed like this

Reply:
I will remove 
(char *) out_info.data)[len] = '\0';  line as @jaeyun-jung  recommanded
and without this, I tested gst-launch cmd, and it works properly